crest/Assets/Crest/Crest/Scripts/Shapes/ShapeGerstner.cs

```diff
@@ -106,6 +106,10 @@ protected static OceanWaveSpectrum DefaultSpectrum
             }
         }
 
+#if UNITY_EDITOR
+        internal bool _isPrefabStageInstance = false;
```

Comment:
i assume it was not possible to have centralised state for this, like share the one on the OR or a static flag or something? sorry if you already mentioned it, i'm in "review mode" of github now and i dont want to risk clicking away :)

Reply:
Nah. Unfortunately each one needs to track it themselves.